From 0081008d4f1bd84b5baf090c4801ffb3609a8b8e Mon Sep 17 00:00:00 2001
From: Jim Bride <jim.bride@intel.com>
Date: Fri, 10 Feb 2012 09:41:56 -0800
Subject: Add lsusb to the list of supported utilities.

Add lsusb to the list of supported utilities so a symlink to
toolbox named lsusb is generated in /system/bin.

Issue: ADEV-21
Change-Id: I1012d31f5e5ddcc6f5d670c6f94951e06a9cf500
Category: Device Enablement
Domain: AOSP-toolbox
Upstream-candidate: Yes
Signed-off-by: Jim Bride <jim.bride@intel.com>
---
 toolbox/Android.mk |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/toolbox/Android.mk b/toolbox/Android.mk
index 40a7939..d79b0d0 100644
--- a/toolbox/Android.mk
+++ b/toolbox/Android.mk
@@ -58,6 +58,7 @@ TOOLS := \
 	du \
 	md5 \
 	clear \
+	lsusb \
 	getenforce \
 	setenforce \
 	chcon \
-- 
1.7.4.1

